shell/browser/ui/views/autofill_popup_view.cc

```diff
@@ -224,7 +224,8 @@ void AutofillPopupView::DoUpdateBoundsAndRedrawPopup() {
   if (!popup_)
     return;
 
-  GetWidget()->SetBounds(popup_->popup_bounds_);
+  GetWidget()->SetBounds(popup_->popup_bounds_.Union(
+      gfx::Rect(popup_->popup_bounds_.origin(), gfx::Size(1, 1))));
```

Comment:
If `popup_->popup_bounds_` could be empty, then so would be `popup_->popup_bounds_in_view()`, which is used in some other places.